docs/token.html

```diff
@@ -0,0 +1,300 @@
+              <code>github-app-installation-token-action</code> step at the beginning of your workflow, as shown in this
+              example.
+            </p>
+          </div>
+          <div class="p-2 full-width">
+            <div id="example" class="card">
+              <pre id="example-data">
+- id: auth
+  uses: jnwng/github-app-installation-token-action@v1
+  with:
+    appId: ${{ secrets.UHA_APP_ID }}
+    installationId: ${{ secrets.UHA_INSTALLATION_ID }}
+    privateKey: ${{ secrets.UHA_PRIVATE_KEY }}
+- uses: actions/checkout@master
+- name: Apply herald rules
+  uses: gagoar/use-herald-action@v2.0.0
+  with:
+    GITHUB_TOKEN: ${{ steps.auth.outputs.token }}
+    rulesLocation: herald_rules/*.json</pre
+              >
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="container parting">
+      <hr />
+      <p>
+        That's it!<br />
+        Go make amazing things in your organization,<br />
+        with the help of <code>use-herald-action</code>!
+      </p>
+      <p>
+        Have feedback or questions about this installation flow?<br />
+        Open an issue <a href="https://github.com/gagoar/use-herald-action/issues">here</a>!
+      </p>
+    </div>
+  </div>
+
+  <footer>
+    This site was made with &lt;3 by <a href="https://github.com/cyamonide" target="_blank">cyamonide</a>.
+  </footer>
+
+  <script
+    src="https://code.jquery.com/jquery-3.5.1.slim.min.js"
+    integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
+    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
+    crossorigin="anonymous"
+  ></script>
+  <script
+    src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"
+    integrity="sha384-B4gt1jrGC7Jh4AgTPSdUtOBvfO8shuf57BaghqFfPlYxofvL8/KUEfYiJOMMV+rV"
+    crossorigin="anonymous"
+  ></script>
+</body>
+
+<script>
+  // Function to transition the page to a validated state
+  const pageValidated = () => {
+    document.getElementById('header-text').innerHTML = 'Your app has been validated!';
+    document.getElementById('steps-parent').style.display = 'block';
+    setTimeout(() => {
+      document.getElementById('steps-parent').style.opacity = '1';
+    }, 100);
+
+    document.title = 'UHA - Post-setup instructions';
+  };
+
+  // Function to transition the page to a failed state
+  const pageFailed = () => {
+    document.getElementById('header-text').innerHTML = 'Something went wrong...<br />Give that another go, will ya?';
+
+    document.title = 'UHA - Validation failed';
+  };
+
+  document.getElementById('steps-parent').style.display = 'none';
+  document.getElementById('steps-parent').style.opacity = '0';
+
+  // Get URL parameters
+  const { code } = getParams(window.location.href);
```

Comment:
I haven't done anything in the browser for so long, that I didn't know you could do Destructuring assignment.

Reply:
I was surprised too - I had to try it, and expected it to fail, but it didn't!